src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/PackageReferenceProject.cs

```diff
@@ -245,42 +264,36 @@ public virtual async Task<ProjectPackages> GetInstalledAndTransitivePackagesAsyn
         /// <summary>
         /// Get All Installed packages that transitively install a given transitive package in this project
         /// </summary>
-        /// <param name="transitivePackage">Identity of given transtive package</param>
+        /// <param name="installedPackages">The list of installed packages</param>
+        /// <param name="targetsList">The list of targets</param>
         /// <param name="ct">Cancellation Token</param>
         /// <returns>A dictionary, indexed by Framework/Runtime-ID with all top (installed)
-        /// packages that depends on given transitive package, or <c>null</c> if none found</returns>
+        /// packages that depends on given transitive package</returns>
         /// <remarks>Computes all transitive origins for each Framework/Runtime-ID combiation. Runtime-ID can be <c>null</c>.
         /// Transitive origins are calculated using a Depth First Search algorithm on all direct dependencies exhaustively</remarks>
-        internal TransitiveEntry GetTransitivePackageOrigin(PackageReference transitivePackage, List<PackageReference> installedPackages, IReadOnlyList<LockFileTarget> targetsList, CancellationToken ct)
+        static internal Dictionary<string, TransitiveEntry> ComputeTransitivePackageOrigins(List<PackageReference> installedPackages, IReadOnlyList<LockFileTarget> targetsList, CancellationToken ct)
```

Comment:
Do you think ordering should match to

public static void ....

internal static void

```suggestion
        internal static Dictionary<string, TransitiveEntry> ComputeTransitivePackageOrigins(List<PackageReference> installedPackages, IReadOnlyList<LockFileTarget> targetsList, CancellationToken ct)
```

Reply:
I completely agree. Thanks!